v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/utils/FileSystemUtils.java

```diff
@@ -0,0 +1,51 @@
+package com.google.cloud.teleport.v2.neo4j.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import org.apache.beam.sdk.io.FileSystems;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utilities for reading local and remote file resources.
+ */
+public class FileSystemUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(FileSystemUtils.class);
+
+    public static String getPathContents(String gsPath) {
+        StringBuilder textBuilder = new StringBuilder();
+
+        try {
+            ReadableByteChannel chan = FileSystems.open(FileSystems.matchNewResource(
+                    gsPath
+                    , false /* is_directory */
```

Comment:
Seems incorrectly formatted. Is this what spotless generated?

Reply:
Looks good now